### ISSUE TITLE
[Code Hints] Code hinting should be disabled in regexps

### ISSUE THREAD

**Issue by nicolo-ribaudo** [adobe/brackets#10160]
While I write regular expressions, Brackets shows js variables:

![regex-code-hints](https://cloud.githubusercontent.com/assets/7000710/5414455/620b752c-821f-11e4-8bf0-59b6ee72ba7f.jpg)

**Comment by peterflynn** [comment]
Clearing milestone since it's marked low priority and has been around for a long time.  Fix is already in progress though.

**Comment by nethip** [comment]
@peterflynn Is anyone looking at fixing this? Is it the fix on a branch? Or should we put a name on this task?

**Comment by redmunds** [comment]
@nethip This is fixed in PR https://github.com/adobe/brackets/pull/10242 -- if you look back through the history of this issue you can see the link. It's just waiting on a unit test.

**Comment by nethip** [comment]
@redmunds Thanks for clarifying! I will do some unit testing and if things look good let us merge this.

**Comment by nethip** [comment]
@prksingh Could you take a look at this and do some unit testing around this area. And also please review PR #10242 .

**Comment by prksingh** [comment]
@nethip I will add the unit tests and update the PR
